lang/rust/Makefile

```diff
@@ -22,7 +23,6 @@ PKG_HOST_ONLY:=1
 
 include $(INCLUDE_DIR)/host-build.mk
 include $(INCLUDE_DIR)/package.mk
-include ./rust-values.mk
```

Comment:
Why that?

Reply:
It was moved up, because:

> `HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/rustc-$(PKG_VERSION)-src-$(RUSTC_TARGET_ARCH)`

needs the `RUSTC_TARGET_ARCH` variable, which is provided by `./rust-values.mk`